eksctl/template.jsonnet

```diff
@@ -1,4 +1,14 @@
-// Exports an eksctl config file for carbonplan cluster
+// This file is a jinja2 template of a jsonnet template of a eksctl's cluster
+// configuration file, which is in turn can be used with the `eksctl` CLI to both
+// update and initialize a AWS EKS based cluster.
+//
+// This jinja2 template is only used by the deployer script as part of creating
+// new clusters. If a relevant change is made here or the dependent file
+// libsonnet/nodegroup.jsonnet, one may consider if we should manually update
+// already generated jsonnet files in this folder.
```

Comment:
For reference, this is the comment that I updated to clarify the situation about this being a jinja2 template of a jsonnet template.